src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -236,6 +237,23 @@ def step_index(self):
         """
         return self._step_index
 
+    @property
```

Comment:
you didn't make these changes, no? 
why are they showing up here?

Reply:
That's strange. Because this is not different from the current main branch in diffusers. So it isn't a change at all.